### DISCUSSION TITLE
AI警察

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#620] [General]
他們用ai犯罪，我們也用ai破案，ai真是神奇
https://www.bnext.com.tw/article/73519/chainsight-jan-mag

**Comment by chia-kai-chang** [comment]
「神奇」這個字眼太玄學，請嘗試使用科學一點的詞彙

**Comment by titan92601** [comment]
AI 原是用來推進人類發展的，如今盡然優先發展出犯罪才想出應對辦法，那我們是不是反被ai操控呢 令人加以深思啊～～

**Comment by larry40305** [comment]
畢竟AI發明本來就是給人類用的，所以不管人類拿去做什麼都有可能。

**Comment by magicmicheal** [comment]
一開始看到AI警察的標題，還有一些留言，讓我馬上聯想到，一些電影裡提到的一個議題，那就是藉由機器人當法官，這項職業在電影中是完全被圍來世界禁止嫆雷給機器人擔任的，原因是機器人無法給人類定罪，不知道各位對於這件事有什麼看法

**Comment by manipulateed** [comment]
能自動幫忙辨別是否詐騙真的很有利於判斷能力較不足的人可以有更客觀的數據去提醒自己要警惕一些，希望之後也能運用在電話或是訊息詐騙的層面，讓犯罪大大減少。

**Comment by boson13579** [comment]
雖然運用AI幫助警察們破案能解決大部分詐騙，其根源還是要剷除才較保險。多多教育和宣傳防詐，尤其是小孩及年長者，才能真正減少被詐騙的受害者。
